### PR TITLE
Disable URL editing after article is published.

### PR DESCRIPTION
**Pull request by ourmaninamsterdam**
screen shot 2015-03-25 at 14 02 49

@tomas-stefano and I will pair on some acceptance tests in the near future when availability allows...

TP#4964

### CODE REVIEW

File: app/models/comfy/cms/page.rb [hunk 1]
```diff
@@ -78,4 +78,8 @@ def suppress_mirrors_from_links_recirculation
       mirror.update_column(:suppress_from_links_recirculation, suppress_from_links_recirculation)
     end
   end
+
+  def has_ever_been_published?
+    revisions.map {|revision| revision.data[:event]}.include?("published")
```
Comment (tomas-stefano) [inline app/models/comfy/cms/page.rb]: Double quotes

File: app/models/comfy/cms/page.rb [hunk 2]
```diff
@@ -78,4 +78,8 @@ def suppress_mirrors_from_links_recirculation
       mirror.update_column(:suppress_from_links_recirculation, suppress_from_links_recirculation)
     end
   end
+
+  def has_ever_been_published?
```
Comment (tomas-stefano) [inline app/models/comfy/cms/page.rb]: You can remove the has_ prefix.

**Comment by ourmaninamsterdam** [comment]
@tomas-stefano @dougdroper Changes made

**Comment by tomas-stefano** [comment]
:+1:

**Pull request merged**